doc/stdlib/base.rst

```diff
@@ -734,90 +418,60 @@ Types
 .. function:: super(T::DataType)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       super(T::DataType)
 
    Return the supertype of DataType T
 
 .. function:: issubtype(type1, type2)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       issubtype(type1, type2)
 
    True if and only if all values of ``type1`` are also of ``type2``\ . Can also be written using the ``<:`` infix operator as ``type1 <: type2``\ .
 
 .. function:: <:(T1, T2)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
 
-       issubtype(type1, type2)
-
-   True if and only if all values of ``type1`` are also of ``type2``\ . Can also be written using the ``<:`` infix operator as ``type1 <: type2``\ .
+   Subtype operator, equivalent to ``issubtype(T1,T2)``\ .
 
 .. function:: subtypes(T::DataType)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       subtypes(T::DataType)
 
    Return a list of immediate subtypes of DataType T. Note that all currently loaded subtypes are included, including those not visible in the current module.
 
 .. function:: typemin(type)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       typemin(type)
 
    The lowest value representable by the given (real) numeric type.
 
 .. function:: typemax(type)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       typemax(type)
 
    The highest value representable by the given (real) numeric type.
 
 .. function:: realmin(type)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       realmin(type)
 
    The smallest in absolute value non-subnormal value representable by the given floating-point type
 
 .. function:: realmax(type)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       realmax(type)
 
    The highest finite value representable by the given floating-point type
 
 .. function:: maxintfloat(type)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       maxintfloat(type)
 
    The largest integer losslessly representable by the given floating-point type
 
 .. function:: sizeof(type)
 
    .. Docstring generated from Julia source
-   .. code-block:: julia
-
-       sizeof(type)
 
    Size, in bytes, of the canonical binary representation of the given type, if any.
 
```

Comment:
The code block below (`sizeof(s::AbstractString)`) should be another signature of `sizeof` instead of a code block.